app/views/shared/_head.html.erb

```diff
@@ -8,3 +8,5 @@
 <%= render "shared/head/backward_ie_compatibility" %>
 <%= render "shared/head/stylesheets" %>
 <%= render "shared/head/javascripts" %>
+
+<%= yield :head %>
```

Comment:
Can you move this into `/shared/head/_link_tags.html.erb` please?

Otherwise looks good.